TypeInjections/Tool/Tool.fs

```diff
@@ -7,12 +7,48 @@ open LibGit2Sharp
 open System.IO
 open System
 open TypeInjections
+open CommandLine
 
 module Tool =
+    
+    
+    [<Verb("local", HelpText = "[old project path] [new project path] [outputPath]")>]
+    type LocalOptions =
+        { [<Option('o', "old", Required = false, HelpText = "folder path of old project")>]
+          oldFolder: string
+          [<Option('n', "new", Required = false, HelpText = "folder path of new project")>]
+          newFolder: string
+          [<Option('p', "output", Required = false, HelpText = "folder path to place typeCart files")>]
+          outputFolder: string
+          [<Option('f', "file", Required = false, HelpText = "specify filename you want to run typecart on")>]
+          file: seq<string> }
+
+    [<Verb("remote", HelpText = "[github repo url] [new commit] [old commit] [outputPath]")>]
```

Comment:
why does this say `github`, can't it be an arbitrary git server?

Reply:
I know it can clone public repositories hosted on `github` `bitbucket` and `gitlab`. I didn't want people trying to use commands like`ssh://git.amazon.com/pkg/Yucca`

Would a better message be `[https repo url]`?

Reply:
just `git URL`